plugins/forcetorque/src/ForceTorqueDriver.cpp

```diff
@@ -139,6 +142,67 @@ int GazeboYarpForceTorqueDriver::calibrateChannel(int /*ch*/, double /*v*/)
     return AS_OK;
 }
 
+// SIX AXIS FORCE TORQUE SENSORS
+size_t GazeboYarpForceTorqueDriver::getNrOfSixAxisForceTorqueSensors() const
+{
+    return 1;
+}
+
+yarp::dev::MAS_status GazeboYarpForceTorqueDriver::getSixAxisForceTorqueSensorStatus(size_t sens_index) const
+{
+    if (sens_index > 1)
```

Comment:
Hi @traversaro 

Just to double-check about this condition.. I checked about `sens_index`, and in https://www.yarp.it/v3.7/classyarp_1_1dev_1_1ISixAxisForceTorqueSensors.html it says

```
[in] | sens_index | The index of the specified sensor (should be between 0 and getNrOfSixAxisForceTorqueSensors()-1).
```

So in our case only `0` is acceptable if I am not mistaken.

Reply:
Yes, good catch!

Reply:
Fixed in https://github.com/robotology/gazebo-yarp-plugins/pull/628/commits/0662a7a77de0c1baa5e39f8d7957b05314bb9033 .